keyboards/jopr/keymaps/default/keymap.c

```diff
@@ -0,0 +1,59 @@
+    ),
+    [_KANA] = LAYOUT(
+		_______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______,   _______, _______, _______, _______, \
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,           _______, _______, _______, _______, \
+		_______,     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, \
+		_______,       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,              _______, _______, _______,            _______, _______, _______, \
+		_______,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                  _______, _______, _______, _______,   _______, _______, _______, \
+		_______,   _______,   _______,   _______,                                                 _______,   _______,   _______,   _______,     _______,          _______,            _______, _______, _______ \
+    ),
+    [_FN] = LAYOUT(
+		_______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______, _______, _______, _______,   _______,   KC_SYSREQ, _______, _______, _______, \
+		_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,           _______, _______, _______, _______, \
+		_______,     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,       KC_MSTP, KC_VOLU, _______, _______, \
+		_______,       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,              KC_MPRV, KC_MPLY, KC_MNXT,            _______, _______, _______, \
+		_______,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                  KC_MUTE, KC_VOLD, _______, _______,   KC_MSTP, KC_VOLU, KC_MPLY, \
+		_______,   _______,   _______,   _______,                                                 _______,   _______,   _______,   _______,     _______,          _______,            KC_MPRV, KC_VOLD, KC_MNXT \
+    )
+};
+
+void led_set_user(uint8_t usb_led) {
+
+    /*if (usb_led & (1 << USB_LED_KANA)) {
+        DDRF |= (1 << 4); PORTF &= ~(1 << 4);
+    } else {
+        DDRF &= ~(1 << 4); PORTF &= ~(1 << 4);
+    }*/
+
+    if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
+		DDRF |= (1 << 1); PORTF &= ~(1 << 1);
+	} else {
+		DDRF &= ~(1 << 1); PORTF &= ~(1 << 1);
+    }
+
+    if (usb_led & (1 << USB_LED_SCROLL_LOCK)) {
+		DDRF |= (1 << 0); PORTF &= ~(1 << 0);
+	} else {
+		DDRF &= ~(1 << 0); PORTF &= ~(1 << 0);
+    }
+}
+void led_set_keymap(uint8_t usb_led) {
+    if (!(usb_led & (1<<USB_LED_NUM_LOCK))) {
+        register_code(KC_NUMLOCK);
```

Comment:
You can replace these two lines with `tap_code(KC_NUMLOCK)`, and if you have `TAP_CODE_DELAY` defined, it will wait that many milliseconds before unregistering.